### ISSUE TITLE
Revise the (whole) code in general

### ISSUE THREAD

**Issue by urbanware-org** [urbanware-org/salomon#31]
Many temporary variables in the code are unnecessary and also do not have a meaningful name in some cases. So, they should be removed by replacing code like this

```bash
for num in $(seq 10); do
    temp="$foo $num"
    foo=$temp
done
```

with that:

```bash
for num in $(seq 10); do
    foo="$foo $num"
done
```

**Comment by urbanware-org** [comment]
Created the branch [remove-temp-variables](https://github.com/urbanware-org/salomon/tree/remove-temp-variables) where all `temp` variables have either been removed or (where necessary) renamed accordingly based on their intended use.

The branch has not been tested extensively so far, but it already successfully passed a testrun via `testrun.sh`.

**Comment by urbanware-org** [comment]
Created the branch [remove-temp-variables](https://github.com/urbanware-org/salomon-bsd/tree/remove-temp-variables) where all `temp` variables have either been removed or (where necessary) renamed accordingly based on their intended use.

The branch has not been tested, yet. Performing a testrun via `testrun.sh` is still outstanding.

**Comment by urbanware-org** [comment]
Branch has been merged.

**Comment by urbanware-org** [comment]
**Fixed** in version **1.16.3**.
